drivers/sensor/ak8975/ak8975.c

```diff
@@ -55,7 +55,7 @@ static void ak8975_convert(struct sensor_value *val, int16_t sample,
 	conv_val = sample * AK8975_MICRO_GAUSS_PER_BIT *
 		   ((uint16_t)adjustment + 128) / 256;
 	val->val1 = conv_val / 1000000;
-	val->val2 = conv_val % 1000000;
```

Comment:
Seems like this was changed in a different file than the akm09918c file which is being added?

Reply:
Oops :)